boot.php

```diff
@@ -486,6 +486,16 @@
  */
 define('NULL_DATE', '0001-01-01 00:00:00');
 
+/**
+ * Deliver commands
+ */
+define('DELIVER_MAIL',       'mail');
```

Comment:
Please don't add any more global constants to the `boot.php` file. Since they are used only in Worker\Delivery, please move them in the class.